demos/twa-offline-first/build.gradle

```diff
@@ -0,0 +1,39 @@
+plugins {
```

Comment:
nit: please, add license headers to the files and ensure all files finish with an empty line.